authors/settings.py

```diff
@@ -78,6 +78,18 @@
 # Database
 # https://docs.djangoproject.com/en/1.11/ref/settings/#databases
 
+if 'TRAVIS' in os.environ:
+    DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'athenatraviscidb',
+        'USER': 'postgres',
```

Comment:
Please pick these from the environment.

Reply:
Thanks for the feedback, let me get to it immediately.